rss/routes.py

```diff
@@ -0,0 +1,28 @@
+"""URL routes for RSS feeds"""
+
+from flask.json import jsonify
+from flask import Blueprint
+from werkzeug.exceptions import NotFound
+from arxiv import status
+
+blueprint = Blueprint('rss', __name__, url_prefix='')
+
+@blueprint.route('/rss', methods=['GET'])
```

Comment:
Minor style point: two blank lines above top-level function and class declarations. We can set up Travis to do linting automatically, and it will point out these kinds of things. More info: https://cul-it.github.io/arxiv-arxitecture/crosscutting/development_practices.html#code-quality-linting